server/controllers/article.controller.js

```diff
@@ -40,5 +40,31 @@ class ArticleController {
     }
     return Response(res, 201, successMessage, { article });
   }
+
+  static async getAllArticles(req, res) {
+    /**
+     * @description - Controller to get all articles endpoint
+     *
+     * @static
+     * @param {object} request
+     * @param {object} response
+     * @returns {object}
+     * @memberof getAllArticles
+    */
+    const successMessage = 'Articles successfully retrieved';
+    const errorMessage = 'There is no article in database';
+    const articles = await Article.findAndCountAll({
+      where: { published: true },
+      include: [{
+        model: User,
```

Comment:
You have to load an article with the corresponding comments. Kindly implement this